testhiml/health/azure/test_himl.py

```diff
@@ -564,12 +603,17 @@ def test_invoking_hello_world_no_config(run_target: RunTarget, tmp_path: Path) -
         'body': 'print(f"The message was: {args.message}")'
     }
     extra_args = [f"--message={message_guid}"]
-    output = render_and_run_test_script(tmp_path, run_target, extra_options, extra_args, run_target == RunTarget.LOCAL)
     expected_output = f"The message was: {message_guid}"
     if run_target == RunTarget.LOCAL:
+        output = render_and_run_test_script(tmp_path, run_target, extra_options, extra_args,
+                                            run_target == RunTarget.LOCAL,
+                                            suppress_config_creation=run_target == RunTarget.AZUREML)
```

Comment:
suppress_config_creation is always False here, and that's the default, so maybe remove this line?

Reply:
Done